src/DynamoCore/Core/DynamoMigrator.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Xml.Serialization;
+using Dynamo.Models;
+using ProtoCore.Lang;
+
+namespace Dynamo.Core
+{
+    public struct FileVersion
+    {
+        public readonly int MajorPart;
+        public readonly int MinorPart;
+
+        public FileVersion(int majorPart, int minorPart)
+        {
+            MajorPart = majorPart;
+            MinorPart = minorPart;
+        }
+    }
```

Comment:
We have had team that updated `Point(x, y, z)` and did a copy-paste, ended up with assigning wrong fields to data member before, because there wasn't unit test to catch editing mistakes. So here's a test I'd recommend:

```csharp
var fileVersion = new FileVersion(12, 34);
Assert.AreEqual(12, fileVersion.MajorPart);
Assert.AreEqual(34, fileVersion.MinorPart);
```